.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,56 @@
+name: Benchmarks
+on:
+  schedule:
+    - cron: "0 0 * * 0-6"  # Run daily at 0am
+  pull_request:
+
+concurrency:
+  group: benchmarks-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+    # If needed, this workflow will create a cached benchmark reference.
+    # This reference will be used for comparative analyses with all PRs benchmarks.
+    reference-benchmarks:
+      if: github.event_name == 'schedule' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:run-benchmarks'))
+
+      uses: ./.github/workflows/interface-unit-tests.yml
+      with:
+        branch: add-pytest-benchmarks
+        pipeline_mode: 'reference-benchmarks'
+
+        # Run a 'lightened' version of the CI on Pull Requests by default
+        # Unless the label `ci:run-full-test-suite` is attached to the PR.
+        # Always runs the full suite for push events.
+        run_lightened_ci: >-
+          ${{
+            github.event_name == 'pull_request'
+            && !contains(github.event.pull_request.labels.*.name, 'ci:run-full-test-suite')
+            || false
+          }}

```

Comment:
This block will also run a "lightened" version for benchmarks (even when it is run on schedule) ... is this intended? 

Lightened = only run against Python 3.9 instead across multiple versions)

Reply:
Yes, this is intended.